packages/core/strapi/lib/commands/transfer/export.js

```diff
@@ -11,92 +11,123 @@ const { isObject, isString, isFinite, toNumber } = require('lodash/fp');
     process.exit(1);
   }
-  const filename = opts.file;
-
-  /**
-   * From local Strapi instance
-   */
-  const sourceOptions = {
-    async getStrapi() {
-      const appContext = await strapi.compile();
-      const app = strapi(appContext);
-
-      app.log.level = 'error';
-
-      return app.load();
-    },
-  };
-  const source = createLocalStrapiSourceProvider(sourceOptions);
-
-  const file = isString(filename) && filename.length > 0 ? filename : getDefaultExportName();
 
-  /**
-   * To a Strapi backup file
-   */
-  const maxSize = isFinite(toNumber(opts.maxSize))
-    ? toNumber(opts.maxSize) * BYTES_IN_MB
-    : undefined;
-
-  const maxSizeJsonl = isFinite(toNumber(opts.maxSizeJsonl))
-    ? toNumber(opts.maxSizeJsonl) * BYTES_IN_MB
-    : undefined;
+  const source = createSourceProvider();
+  const destination = createDestinationProvider(opts);
 
-  const destinationOptions = {
-    file: {
-      path: file,
-      maxSize,
-      maxSizeJsonl,
-    },
-    encryption: {
-      enabled: opts.encrypt,
-      key: opts.key,
-    },
-    compression: {
-      enabled: opts.compress,
-    },
-  };
-  const destination = createLocalFileDestinationProvider(destinationOptions);
-
-  /**
-   * Configure and run the transfer engine
-   */
-  const engineOptions = {
+  const engine = createTransferEngine(source, destination, {
     strategy: 'restore', // for an export to file, strategy will always be 'restore'
-    versionMatching: 'minor', // for an export to file, versionMatching will always be skipped
-    exclude: opts.exclude,
-  };
-  const engine = createTransferEngine(source, destination, engineOptions);
+    versionMatching: 'ignore', // for an export to file, versionMatching will always be skipped
+    transforms: {
+      entities: [
+        {
+          filter(entity) {
+            return !DEFAULT_IGNORED_CONTENT_TYPES.includes(entity.type);
+          },
+        },
+      ],
+    },
+  });
 
   try {
     logger.log(`Starting export...`);
 
     const results = await engine.transfer();
+    const outFile = results.destination.file.path;
+
     const table = buildTransferTable(results.engine);
     logger.log(table.toString());
 
-    if (!fs.pathExistsSync(results.destination.file.path)) {
-      logger.log(file);
-      throw new Error('Export file not created');
+    const outFileExists = await fs.pathExists(outFile);
+    if (!outFileExists) {
+      throw new Error(`Export file not created "${outFile}"`);
     }
 
     logger.log(`${chalk.bold('Export process has been completed successfully!')}`);
-    logger.log(`Export archive is in ${chalk.green(results.destination.file.path)}`);
+    logger.log(`Export archive is in ${chalk.green(outFile)}`);
     process.exit(0);
   } catch (e) {
     logger.error('Export process failed unexpectedly:', e.toString());
     process.exit(1);
   }
 };
+
+/**
+ * It creates a local strapi destination provider
+ */
+const createSourceProvider = () => {
+  let strapi = null;
```

Comment:
Instead of storing it here, can we store it in the export command? Even if we had multiple local source providers, they could still share the same Strapi rather than creating their own. And that way we also have access to it for the command to send telemetry events through it.